server/studyspaces.py

```diff
@@ -0,0 +1,90 @@
+from flask import request, jsonify
+from .base import *
+from server import app
+from bs4 import BeautifulSoup
+import urllib2
+import requests
+
+def getIDs(): # extracts the ID's of the room into groupStudyCodes
+	groupStudyCodes = []
+	url = "http://libcal.library.upenn.edu/booking/vpdlc"
+	soup = BeautifulSoup(urllib2.urlopen(url).read(), 'lxml')
+	l = soup.find_all('option')
+	for element in l:
+		if element['value'] != '0':
+			url2 = "http://libcal.library.upenn.edu" + str(element['value'])
+			soup2 = BeautifulSoup(urllib2.urlopen(url2).read(), 'lxml')
+			id = soup2.find('input', attrs={"id" : "gid"})['value']
+			newDict = {} 
+			newDict['id'] = int(id)
+			newDict['name'] = element.contents[0]
+			groupStudyCodes.append(newDict)
+	return groupStudyCodes
+
+def getIDDict(): # extracts the ID's of the room into a dictionary
```

Comment:
Same comments as in `getID()`. Also, when using similar function names, please document the difference, as I have no idea what the difference between these 2 functions is.